DataManagementSystem/Agent/FTSAgent.py

```diff
@@ -925,7 +925,10 @@ def __finalizeFTSJob( self, request, ftsJob ):
     if not processFiles["OK"]:
       log.error( processFiles["Message"] )
       return processFiles
-    ftsFilesDict = self.updateFTSFileDict( ftsFilesDict, processFiles["Value"] )
+    processFiles = processFiles['Value']
+    if processFiles['toRegister']:
+      log.error( "%f files could not be registered in FC" % len( processFiles['toRegister'] ) )
```

Comment:
OK, but then every developer should go through the code as I can tell you that there are very many non-static first part!!!!
May be it is worth a reminder to all developers: every time you touch a module, fix the logging...

Le 20 janv. 2015 à 00:39, Andrei Tsaregorodtsev <notifications@github.com> a écrit :

> In DataManagementSystem/Agent/FTSAgent.py:
> 
> > @@ -925,7 +925,10 @@ def __finalizeFTSJob( self, request, ftsJob ):
> >      if not processFiles["OK"]:
> >        log.error( processFiles["Message"] )
> >        return processFiles
> > -    ftsFilesDict = self.updateFTSFileDict( ftsFilesDict, processFiles["Value"] )
> > +    processFiles = processFiles['Value']
> > +    if processFiles['toRegister']:
> > +      log.error( "%f files could not be registered in FC" % len( processFiles['toRegister'] ) )
> The Magic DB is the SystemLoggingDB where all the entries are classified by the static message part which is the first argument. The variable part is printed in the log, so you can print the number of files there. Make it so, that it makes sense for the reader and does not screw up SystemLogging at the same time.
> 
> —
> Reply to this email directly or view it on GitHub.
> 

-- 
Dr Philippe Charpentier
J09210, Physics Department, CERN, CH-1211 Genève 23
LHCb Experiment Distributed Computing Coordinator
Office 2-R-007, Tel : +41 22 767 4244 ,  GSM : +41 76 487 0167
mailto:Philippe.Charpentier@cern.ch  http://cern.ch/Philippe.Charpentier